src/main/java/view/ChatPageUI/ChatWindow.java

```diff
@@ -19,13 +19,14 @@
 
 public class ChatWindow extends JFrame {
     private final ChatController chatController;
-    private final String currentUser = "jason";
+    private final String currentUser;
```

Comment:
Uses the correct logic where the username should be the user that is currently logged in, not a placeholder.